python/pyarrow/_flight.pyx

```diff
@@ -1413,6 +1413,10 @@ cdef class ServerCallContext:
         """
         return tobytes(self.context.peer_identity())
 
+    def peer(self):
+        """Get the address of the peer."""
+        return tobytes(self.context.peer())
```

Comment:
Why `tobytes`? I would expect `frombytes`, to get a proper string.

Reply:
Good point - fixed.